Task7.1/Factories/PaperBookLibraryFactory.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Task7.Enitites;
+using Task7.Interfaces;
+
+namespace Task7.Factories
+{
+    internal class PaperBookLibraryFactory : AbstractFactory
+    {
+        public string Path { get; }
+
+        public PaperBookLibraryFactory(string path)
+        {
+            Path = path;
+        }
```

Comment:
Use separate methods to Load necessary part of CSVData, not to parse the whole csv file each time